carapace-server/src/test/java/org/carapaceproxy/server/certificates/CertificatesTest.java

```diff
@@ -160,6 +163,54 @@ public void test() throws Exception {
         assertTrue(data.isManual());
     }
 
+    @Test
+    @Parameters({"acme", "manual"})
+    public void testUploadTypedCertificate(String type) throws Exception {
+        configureAndStartServer();
+        int port = server.getLocalPort();
+        DynamicCertificatesManager dynCertsMan = server.getDynamicCertificateManager();
+
+        // Uploading certificate without data (for type="acme" means creating an order for ACME certificate)
```

Comment:
what happens if the client request the creation of an existing domain ?
is it possible to change the 'type' ?